Deeploy/FutureExtension/Bindings/AutoFutureBinding.py

```
@@ -48,14 +49,14 @@ def assignStateReferenceElement(self, ctxt) -> NetworkContext:
                         stateElementCandidates.append(reference)
 
             if len(stateElementCandidates) == 1:
-                print(f"WARNING: Automagically assigning state Element of {self}")
+                log.warninging(f"Automagically assigning state Element of {self}")
                 for key, value in operatorRepresentation.items():
```

Ruff (0.12.2)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)